contrib/docker/builder/Dockerfile

```diff
@@ -9,7 +9,7 @@ RUN apt-get update && apt-get install -y \
     autoconf automake bsdmainutils ccache cmake curl g++ g++-mingw-w64-x86-64 gcc gcc-mingw-w64-x86-64 git \
     libboost-all-dev libbz2-dev libcap-dev libdb4.8-dev libdb4.8++-dev libevent-dev libminiupnpc-dev libprotobuf-dev \
     libqrencode-dev libssl-dev libtool libzmq3-dev make pkg-config protobuf-compiler python-pip qtbase5-dev \
-    qttools5-dev-tools python3-zmq
+    qttools5-dev-tools python3-zmq libminizip-dev
```

Comment:
Ubuntu 14.04 have no `libminizip-dev`.

Reply:
ok, I'm trying the Ubuntu 14 version. After I will squash the commits.

Reply:
Please dont squash after someone push a review that make it review more difficult.

Reply:
Understand, but I also don't want to add a lot of small commits for the same feature. Anyhow, don't think we can push this